packages/caver-rpc/src/governance.js

```diff
@@ -157,15 +157,18 @@ const Governance = function Governance(...args) {
          * @instance
          *
          * @example
+         * const result = await caver.rpc.governance.getChainConfig(0)
+         * // latest by default
          * const result = await caver.rpc.governance.getChainConfig()
          *
+         * @param {number|string} blockNumber A block number, or the hex number string to query chain configuration.
          * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
-         * @return {Promise<object>} The latest chain configuration
+         * @return {Promise<object>} The latest chain configuration, or chain configuration at the given block number.
          */
         new Method({
             name: 'getChainConfig',
-            call: 'governance_chainConfig',
-            params: 0,
+            call: 'governance_getChainConfig',
+            params: 1,
```

Comment:
Please check again if we should delete the governance_chainConfig and klay_chainConfig API. As I know It will be deleted in v1.11.0.

Reply:
*_chainConfig are deleted in 1.10.2 (https://github.com/klaytn/klaytn/pull/1783/files#diff-93a88b2ae3199dd95f4299cfa66e47284a868269c4298641a1f692d1c5dfb2d8L341-L344). Refer to *further comments* section in https://github.com/klaytn/klaytn/pull/1783

Reply:
@nohkwak Sorry for the confusion, *_chainConfig APIs remain in the 1.10.2. However, *_getChainConfig APIs are introduced. *_chainConfig APIs will be deprecated from 1.11.

*_chainConfig
- v1.10.2: O
- v1.11 : X

*_getChainConfig
- v1.10.2: O
- v1.11 : O

So it's recommended to start using *_getChainConfig

Reply:
Then I leave the changes in the PR for now, and will release caver-js after nodes are updated successfully.
Thanks to @kjeom and @ian0371.